config/migrations/20220714120100-insert-internal-x-publication-activities-kanselarij.sparql

```diff
@@ -0,0 +1,107 @@
+        }
+      }
+    }
+  }
+  BIND (IRI(CONCAT("http://themis.vlaanderen.be/id/interne-beslissing-publicatie-activiteit/", ?newuuid)) AS ?decisionPub)
+}
+
+;
+
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?documentPub a ext:InternalDocumentPublicationActivity .
+    ?documentPub mu:uuid ?newuuid .
+    ?documentPub ext:internalDocumentPublicationActivityUsed ?meeting .
+    ?documentPub prov:startedAtTime ?publicationTime .
+    ?documentPub generiek:geplandeStart ?plannedDocPublicationDate .
+  }
+} WHERE {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?meeting a besluit:Vergaderactiviteit .
+    {
+      SELECT DISTINCT ?meeting (STRUUID() AS ?newuuid) ?publicationTime ?plannedDocPublicationDate ?releaseStatus WHERE {
+        GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+          ?meeting a besluit:Vergaderactiviteit ;
+            ext:algemeneNieuwsbrief ?newsletter .
+          ?newsletter ext:issuedDocDate ?plannedDocPublicationDate .
+          OPTIONAL { ?meeting ext:releasedDocuments ?publicationTime . }
+          BIND(IF(BOUND(?publicationTime),
+          <http://themis.vlaanderen.be/id/concept/vrijgave-status/27bd25d1-72b4-49b2-a0ba-236ca28373e5>,
+          <http://themis.vlaanderen.be/id/concept/vrijgave-status/1bd5b05a-07af-46d3-827f-297f993b8b54>)
+            AS ?releaseStatus)
+        }
+      }
+    }
+  }
+  BIND (IRI(CONCAT("http://themis.vlaanderen.be/id/interne-document-publicatie-activiteit/", ?newuuid)) AS ?documentPub)
+}
+
+# TODO KAS-3431 all current themis-publications released?
+# TODO KAS-3431 add themis-publication for recent agendas (not legacy? would be inconsistent with current code). Legacy with status "planned" = not released
+
+;
+
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?themisPublicationActivity a ext:ThemisPublicationActivity ;
+      mu:uuid ?newThemisPublicationuuid ;
+      prov:used ?meeting ;
+      ext:scope "newsitems" ;
+      ext:scope "documents" ;
+      generiek:geplandeStart ?plannedDocPublicationDate ;
+      adms:status <http://themis.vlaanderen.be/id/concept/vrijgave-status/1bd5b05a-07af-46d3-827f-297f993b8b54> .
+  }
+}
+WHERE {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?meeting a besluit:Vergaderactiviteit .
+    {
+      SELECT DISTINCT ?meeting (STRUUID() AS ?newThemisPublicationuuid) ?plannedDocPublicationDate WHERE {
+        GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+          ?meeting a besluit:Vergaderactiviteit ;
+            ext:algemeneNieuwsbrief ?newsletter ;
+            ext:finaleZittingVersie "false"^^<http://mu.semte.ch/vocabularies/typed-literals/boolean> .
+          ?newsletter ext:issuedDocDate ?plannedDocPublicationDate .
+        }
+      }
+    }
+    BIND(IRI(CONCAT("http://themis.vlaanderen.be/id/themis-publicatie-activiteit/", ?newThemisPublicationuuid)) AS ?themisPublicationActivity)
+  }
+}
+
```

Comment:
Ik had deze query gemaakt om voor alle openstaande meetings ook een themis-publication aan te maken.
DIt is vooral voor cypress test data.
In praktijk (release in het weekend) gaat deze query niet de agenda opvangen die al is afgesloten op vrijdag, maar de documenten nog niet zijn vrijgegeven.
Daar moet nog een aparte query voor gemaakt worden, of deze aangepast.